### DISCUSSION TITLE
setpdy.sh?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [NOAA-EMC/global-workflow#2855] [Q&A] [answered]
I've made some progress running on Frontera, but have hit a new snag that I was hoping someone could help with. 

```
++ jjob_header.sh[81]: setpdy.sh
/work2/02441/bcash/frontera/SFS_baseline/ush/jjob_header.sh: line 81: setpdy.sh: command not found
```
I ran this job manually because rocoto is not yet working properly with the container, but some poking around with `find` indicates that I don't have `setpdy.sh` anywhere. Do I need to separately install https://github.com/NOAA-EMC/NCEPLIBS-prod_util in addition to global-workflow?

**Comment by WalterKolczynski-NOAA** [comment, marked as answer]
There should be a `prod_util` module available in the `gsi-addon-env` environment of your spack-stack installation.

**Reply by benjamin-cash** [reply]
Thanks @WalterKolczynski-NOAA - it was available but didn't get loaded.
